src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,90 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
+
+.card {
+    width: 100%;
+    margin: 10px auto;
+    background-color: white;
+    box-shadow: 0px 5px 20px #555;
```

Comment:
@baqianxin this box-shadow will give style conflicts with the card´s in admin UI.
In general i would suggest to set the box-shadow not  as static css value, better a sass var like `$card-box-shadow`.